environments/production/public.yml

```diff
@@ -65,7 +65,7 @@ postgres_s3: True
 
 aws_region: 'us-east-1'
 
-
+postgresql_version: '9.4'
```

Comment:
FYI @NitigyaS I have updated this on all envs (by switching default to 9.6) you should remember update this after you upgrade postgres on prod.

Reply:
on enikshay also you need to set it 9.4 .